### PR TITLE
changed landing and navbar for oss

### PR DESCRIPTION
**Pull request by Abhishek-More**

### CODE REVIEW

**Review by jay-khatri — DISMISSED** [review]
horizantal scroll bug + add github/discord here too?
https://www.loom.com/share/670ca12965cc4b6aac03d754d9b4b9f0

**Review by Vadman97 — APPROVED** [review]

File: pages/index.tsx [hunk 1]
```diff
@@ -192,26 +192,25 @@ const Home: NextPage = () => {
           <div className={styles.heroBugLeft}>
             <Image src={HeroBugLeft} alt="bug left" />
           </div>
-          <div className={styles.heroBugRight}>
-            <Image src={HeroBugRight} alt="bug right" />
```
Comment (Vadman97) [inline pages/index.tsx:196]: why was this removed @Abhishek-More ? was it some problem with the horizontal scrolling? i think we still want this

cc @jay-khatri

Comment (Abhishek-More) [reply pages/index.tsx:196]: Was experimenting with the horizontal scrolling bug, but forgot to add the image back in. Made a new PR to put the image back. https://github.com/highlight/highlight.io/pull/292
